Extension/lib/utils/user-settings.js

```diff
@@ -190,6 +191,14 @@ adguard.settings = (function (adguard) {
         setProperty(settings.DEFAULT_WHITE_LIST_MODE, enabled);
     };
 
+    var isIntegrationDisabled = function () {
```

Comment:
That's weird to check if something is `disabled`.

It'd be better to have `isIntegrationEnabled`/`setIntegrationEnabled(value)` functions